packages/@react-spectrum/actiongroup/stories/ActionGroup.stories.tsx

```diff
@@ -222,6 +233,91 @@ storiesOf('ActionGroup', module)
   .add(
     'with tooltips',
     () => renderTooltips({})
+  )
+  .add(
+    'overflowMode: wrap',
+    () => (
+      <div style={{padding: '10px', resize: 'horizontal', overflow: 'auto', width: 250, backgroundColor: 'var(--spectrum-global-color-gray-50)'}}>
+        <ActionGroup overflowMode="wrap" onAction={action('onAction')}>
+          <Item>
+            <DrawIcon />
+            <Text>Edit</Text>
+          </Item>
+          <Item>
+            <CopyIcon />
+            <Text>Copy</Text>
+          </Item>
+          <Item>
+            <DeleteIcon />
+            <Text>Delete</Text>
+          </Item>
+          <Item>
+            <MoveIcon />
+            <Text>Move</Text>
+          </Item>
+          <Item>
+            <DuplicateIcon />
+            <Text>Duplicate</Text>
+          </Item>
+        </ActionGroup>
+      </div>
+    )
+  )
+  .add(
+    'overflowMode: collapse',
+    () => (
+      <div style={{padding: '10px', resize: 'horizontal', overflow: 'auto', width: 250, backgroundColor: 'var(--spectrum-global-color-gray-50)'}}>
+        {renderCollapsible()}
+        {renderCollapsible({density: 'compact'})}
+        {renderCollapsible({density: 'compact', isJustified: true})}
+        {renderCollapsible({isQuiet: true})}
+      </div>
+    )
+  )
+  .add(
+    'buttonLabelBehavior: hide',
+    () => (
+      <div style={{padding: '10px', resize: 'horizontal', overflow: 'auto', width: 250, backgroundColor: 'var(--spectrum-global-color-gray-50)'}}>
+        {renderCollapsible({buttonLabelBehavior: 'hide'})}
+        {renderCollapsibleText({buttonLabelBehavior: 'hide'})}
+      </div>
+    )
+  )
+  .add(
+    'buttonLabelBehavior: collapse',
+    () => (
+      <div style={{padding: '10px', resize: 'horizontal', overflow: 'auto', width: 500, backgroundColor: 'var(--spectrum-global-color-gray-50)'}}>
+        {renderCollapsible({buttonLabelBehavior: 'collapse'})}
+        {renderCollapsibleText({buttonLabelBehavior: 'collapse'})}
+      </div>
+    )
+  )
+  .add(
+    'overflowMode: collapse, selection',
+    () => (
+      <div style={{padding: '10px', resize: 'horizontal', overflow: 'auto', display: 'flex', gap: 10, width: 300, backgroundColor: 'var(--spectrum-global-color-gray-50)'}}>
+        {renderCollapsibleFormatting({density: 'compact', maxWidth: '50%', isEmphasized: true})}
+        {renderCollapsibleAlignment({density: 'compact', maxWidth: '50%', isEmphasized: true})}
+      </div>
+    )
+  )
+  .add(
+    'overflowMode: collapse, selection',
+    () => (
+      <div style={{padding: '10px', resize: 'horizontal', overflow: 'auto', display: 'flex', gap: 10, width: 300, backgroundColor: 'var(--spectrum-global-color-gray-50)'}}>
+        {renderCollapsibleFormatting({density: 'compact', overflowMode: 'collapse', isEmphasized: true})}
+        {renderCollapsibleAlignment({density: 'compact', overflowMode: 'collapse', isEmphasized: true})}
+      </div>
+    )
+  )
+  .add(
+    'overflowMode: collapse, moreIcon',
```

Comment:
yeah... actually right now it's kinda random. Sometimes the first one collapses first, sometimes the second. It depends on the order the resize observers are fired since they are separate action groups. I think in order to enforce that we'd need to implement something like a Toolbar pattern into which you could place multiple ActionGroups.